contracts/contracts/MessageProcessor.sol

```diff
@@ -296,10 +282,10 @@ contract MessageProcessor is Ownable, SnarkCommon, CommonUtilities, Utilities {
             _coordPubKey,
             _sharedPubKey
         );
-
+        
         require(verifier.verify(_proof, vk, input), "Verification failed");
 
-        return poll.generateNewKeyFromDeactivated(_message, _coordPubKey);
+        return poll.generateNewKeyFromDeactivated(_message, _sharedPubKey);
```

Comment:
the name of this param is misleading. This is not the shared public key, it is the public key of the ecdhKeyPair whose private key is used to generate the shared key with which the message is encrypted. Look at 

function publishMessage(
        Message memory _message,
        PubKey memory _encPubKey
    ) 

in the poll contract for a similar example. I believe we can also use a similar function comment here as the one i poll.publishMessage

Reply:
I did not change the name of the variable but simply the fact that it was passed. Come to think of it, you're right, better to unify it with `_encPubKey` as in the `Pool` method

Reply:
https://github.com/0x3327/maci/pull/33/commits/c51650ffd095574b25026bc7d6944e5386ee5077